Makefile

```diff
@@ -105,18 +105,13 @@ delete-e2e:
 main-in-ci-build:
 	yarn build && \
 	make create-tmp-env-ci && \
-	TMP_ENV_CI=tmp.env.ci docker compose -f docker-compose.yml run --name e2e-tests e2e-tests make start-main-ci || \
-	make del-containers
+	TMP_ENV_CI=tmp.env.ci docker compose -f docker-compose.yml -f docker-compose.override.yml run --name e2e-tests e2e-tests make start-main-ci || \
```

Comment:
did you test this in our CI ?, override is only used locally so if you do this change and works correctly then we dont need an override file

Reply:
it's not used in CI. only in local dev.